cmd/root.go

```diff
@@ -53,7 +54,7 @@ func initConfig(c *cobra.Command, args []string) error {
 		viper.SetConfigFile(configFile)
 	} else {
 		viper.SetConfigName(".klepto")
-		viper.AddConfigPath(homeDir())
+		viper.AddConfigPath(workingDir())
```

Comment:
Why? Isn't the `.` the same as this? 
Also why not home dir? it's pretty useful to have it

Reply:
Using the HomeDir as a default makes the tool harder to work with. Since most projects will store the klepto file inside the project dir which is most of the time the current working dir.